pinot-query-runtime/src/test/resources/queries/Aggregates.json

```diff
@@ -0,0 +1,295 @@
+    "tables": {
+      "tbl": {
+        "schema": [
+          {
+            "name": "int_col",
+            "type": "INT"
+          },
+          {
+            "name": "double_col",
+            "type": "DOUBLE"
+          },
+          {
+            "name": "string_col",
+            "type": "STRING"
+          }
+        ],
+        "inputs": [
+          [
+            2,
+            300,
+            "a"
+          ],
+          [
+            2,
+            400,
+            "a"
+          ],
+          [
+            3,
+            100,
+            "b"
+          ],
+          [
+            100,
+            1,
+            "b"
+          ],
+          [
+            101,
+            1.01,
+            "c"
+          ],
+          [
+            150,
+            1.5,
+            "c"
+          ],
+          [
+            175,
+            1.75,
+            "c"
+          ]
+        ]
+      }
+    },
+    "queries": [
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "comments": "result error:average doesn't work because we round up but h2 round down for integer",
+        "description": "average int",
+        "sql": "SELECT avg(int_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "average double",
+        "sql": "SELECT avg(double_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "average double with filter",
+        "sql": "SELECT avg(double_col) FROM {tbl} WHERE double_col >= 100"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "sum double",
+        "sql": "SELECT sum(double_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "sum int",
+        "sql": "SELECT sum(int_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "max double",
+        "sql": "SELECT max(double_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "max int",
+        "sql": "SELECT max(int_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "count int",
+        "sql": "SELECT count(int_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "count double",
+        "sql": "SELECT count(double_col) FROM {tbl}"
+      },
+      {
+        "psql": "4.2.7",
+        "description": "count, sum group by order by",
+        "sql": "select string_col, count(int_col), sum(double_col) from {tbl} group by string_col order by string_col;"
+      },
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "comments": "parse error",
+        "description": "aggregate having",
+        "sql": "select string_col, count(int_col) from {tbl} a group by string_col order by string_col having exists(select 1 from {tbl} b where count(int_col) = b.int_col);"
+      },
+      {
+        "psql": "4.2.7",
+        "ignored": true,
+        "comments": "plan error",
```

Comment:
can you put more detail planner error here?

Reply:
Done